windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+
+All options available for a given supported command will work as described by the documentation in Certbot for Windows.
+
+## Installation
+
+The following operations needs to be executed using an account with **administrative privileges** on the machine you want to install Certbot.
+
+Certbot can be installed through an installer that is available:
+- on GitHub: go to the Certbot GitHub release (https://github.com/certbot/certbot/releases), select the last version of Certbot (v0.40.0 currently), and download the installer named `certbot-beta-installer-win32.exe in the assets.
+- on Certbot official website: _[specific URL and procedure to be defined]_
+
+The Certbot development team ensures the authenticity of the Windows installer only for these two sources.
+
+Once downloaded, double-click on the installer to run it. You will need to select the location for Certbot installation directory, or keep the default one (`C:\Program Files(x86)\Certbot).
+
+At this point Certbot is ready to run.
+
+## Using Certbot
+
+Certbot is a command line interface with no UI. To interact with it you will need a command line: the classic one (`cmd.exe`) and Powershell aor both supported. You can also use Bash for Windows (available if you installed GIT for Windows for instance) if you prefer.
+
+The command line needs to be executed under **administrative privileges**. Otherwise Certbot will fail to run.
+
+Once the command line is opened, simply write the following command to get the Certbot help:
+```
+certbot --help
+```
+
+Indeed the Certbot installer set up the `PATH` to expose the command line interface `certbot`. So you can run it from any folder, or even include it in your scripts.
+
+In respect with the supported features as described in the _Supported features_ section, you can check the inline help or https://certbot.eff.org/docs/ to check how to use a specific feature in Certbot.
+
+## Configuration
+
+Certbot works essentially with a configuration folder, where all the configuration files and generated certificates are available.
+
+On Windows, this folder is `C:\Certbot`. This can be changed using the `--config-dir` flag on the Certbot CLI.
+
+Within this folder you will find the following folders:
+* `C:\Certbot\live`: contains current versions of issued certificates and keys
+* `C:\Certbot\archive`: contains all current and previous versions of issued certificates and keys
+* `C:\Certificate\logs`: contains Certbot activity logs
+
+Doing backups of `C:\Certbot` folder is everything required to save your Certbot installation.
+
+## Certificates renewals
+
+Renewals are supported by Certbot for Windows and are even already configured. Indeed the Certbot installer created a scheduled task during the installation to execute `certbot renew` twice a day.
+
+All certificates created with Certbot for Windows are then automatically renewed out-of-the-box, providing that required environment to renew is available (eg. credentials, running web server ...).
```

Comment:
provided that the required environment to renew is available

Reply:
e.g.  [abbreviation of the two-word phrase _exempli gratia_]

Reply:
Maybe add something like

> Certbot remembers the method that you used to prove your control over the domain name at the time you originally created each certificate. This method is also used for automated renewals. Therefore, if your environment has changed since you first obtained your certificate (for example, if TCP port 80 is no longer free), you might need to change the associated authentication method in order for automated renewals to continue working.